Applications/Spire/Source/UiViewer/StandardUiProfiles.cpp

```diff
@@ -217,12 +217,18 @@ UiProfile Spire::make_decimal_box_profile() {
           }
         }
       });
+      decimal_box->get_model()->connect_current_signal(
+        [&] (const auto& value) {
+
+        });
       auto current_slot = profile.make_event_slot<QString>(
         QString::fromUtf8("Current"));
       decimal_box->get_model()->connect_current_signal(
-        [=] (const DecimalBox::Decimal& current) {
+        [=, &current] (const DecimalBox::Decimal& value) {
```

Comment:
Can't capture by reference, causes a dangling pointer.

Reply:
In other test profiles we capture the properties that are used in a lambda slot as references because they can't be copied. If current is a reference and its captured as a reference, wouldn't it be valid as long as current was valid?

Reply:
Yes you are right, I mixed up the name of this variable with another.